### DISCUSSION TITLE
Access Denied ke server

### DISCUSSION THREAD

**Discussion by mahircoding** [kirimwa-id/forum#73] [Tanya Jawab]
halo
izin bertanya, sebelumnya sudah berhasil menambahkan device sampai tahap tes kirim pesan berhasil.

Setelah itu, saya coba integrasikan dgn plugin wordpress yang sudah dicustom, dicoba berkali-kali gagal dan akhirnya saya cek ternyata access denied. 

![Screenshot_2021-12-08-06-57-21-49_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/37254341/145124232-a3502f06-29cb-41cf-af4c-83ebd47f9b97.jpg)

Apakah ini kena blokir akibat mencoba integrasi request berkali-kali tapi gagal.

Mungkin ada dari teman-teman yang pernah ngalami atau bisa memberi solusi atas problem ini.

terima kasih

**Comment by rioastamal** [comment]
Yang jadi pertanyaan, error tersebut adalah murni dari konfigurasi di hosting atau tidak? Cek thread #2 untuk tata cara mengajukan pertanyaan agar jawaban to the point dan jelas.
